slice.go

```diff
@@ -240,3 +240,17 @@ func ToMap[K comparable, V any](collection []V, iteratee func(V) K) map[K]V {
 
 	return result
 }
+
+
+// Reject - the opposite of Filter, this method returns the elements of collection that predicate does not return truthy for.
+func Reject[V any](collection []V, predicate func(V, int) bool) []V {
+	result := []V{}
```

Comment:
Wouldn't it be better to implement this as not filter? 

So something like:

```go 
notPredicate := func(val V, i int) bool {
    return !predicate(val, i)
}
return Map(collection, notPredicate)
```

Reply:
Yes!

I won't cost a lot more.